WordPress/src/main/java/org/wordpress/android/ui/LocaleAwareActivity.kt

```diff
@@ -3,53 +3,24 @@ package org.wordpress.android.ui
 import android.content.Context
 import android.content.res.Configuration
 import androidx.appcompat.app.AppCompatActivity
-import org.wordpress.android.ui.prefs.AppPrefs
-import org.wordpress.android.util.LocaleManager
-import org.wordpress.android.util.PerAppLocaleManager
 
 /**
- * Newer versions of the AppCompat library no longer support locale changes at application level,
- * so this activity is used to help handle those changes at activity level.
- * Reference: https://issuetracker.google.com/issues/141869006#comment9
- *
- * All the actual logic is inside the LocaleManager class, which should be used directly in cases where
- * extending from this class is not possible/preferable.
- *
- * Note: please be mindful of the principle of favoring composition over inheritance and refrain from
- * building upon this class unless it's absolutely necessary.
- *
- * Update Dec 2024: We've added experimental support for per-app language preferences which
- * will eventually negate the need for this class. Instead of extending from this class, we
- * should extend from AppCompatActivity once this feature is out of the experimental phase.
+ * Update Dec 2024: We've added support for per-app language preferences which negate
+ * the need for this class. Instead of extending from this class, we should extend
+ * from AppCompatActivity.
```

Comment:
Minor (🔍): This comment can now easily fit in two line, something like:

```
/**
 * Update Dec 2024: We've added support for per-app language preferences which negate the need for this class.
 * Instead of extending from this class, we should extend from AppCompatActivity.
 */
```

Reply:
Good call, but since this activity is being removed in [this PR](https://github.com/wordpress-mobile/WordPress-Android/pull/21523) I'll leave it as-is.